server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -1562,7 +1689,7 @@ public void setDataTTL(long dataTTL) {
 
   @TestOnly
   public List<TsFileResource> getSequenceFileList() {
-    return sequenceFileList;
+    return new ArrayList<>(sequenceFileList);
```

Comment:
This is a method for tests, all its usages are secured. The wrapping is pointless.

Reply:
This if for interface consistency.

Reply:
Well, but it is better to just change the interface instead of introducing unnecessary overhead.